dicom_standard/table_utils.py

```diff
@@ -10,11 +10,28 @@
 from dicom_standard import parse_relations as pr
 
 TableListType = List[List[Tag]]
+StringifiedTableListType = List[List[str]]
 TableDictType = Dict[str, str]
-TableToDictFunc = Callable[[TableListType], List[TableDictType]]
+TableToDictFunc = Callable[[StringifiedTableListType], List[TableDictType]]
 GetTableFunc = Callable[[Tuple[List[TableDictType], Tag]], Dict[str, Any]]
 
 
+def table_to_list(table_div: Tag) -> List[List[str]]:
+    return [[cell.text.strip() for cell in row.find_all('td')]
+            for row in pr.table_rows(table_div)]
+
+
+def get_tables_from_ids(standard: BeautifulSoup, table_ids: List[str], col_titles: List[str]) -> List[TableDictType]:
```

Comment:
I think it could be confusing here that "table to ____" is defined right next to "get tables from _____", but in the first case "table" is the beautiful soup tag and in the second it is a `TableDictType`. This is pretty quickly cleared up by looking at the type signatures (good job keeping these up to date and using clear names!) but it may be worth renaming these slightly to distinguish the two cases.